util/integrations/coordinape/transform.js

```diff
@@ -0,0 +1,48 @@
+import utils from 'web3-utils'
+
+export class CoordinapeEpochTransformError extends Error {
+  constructor(message) {
+    super(message)
+    this.name = 'CoordinapeEpochTransformError'
+  }
+}
+
+const totalGive = contributions =>
+  contributions.reduce((total, { sent, received }) => {
+    if (sent === 0 && received === 0) {
+      // this person either opted out or otherwise did not participate, so we will skip them
+      return total
+    }
+    return total + 100
+  }, 0)
+
+export const transform = async ({ contributions, name }) => {
+  const totalGivePossible = totalGive(contributions)
+  if (totalGivePossible === 0) {
+    throw new CoordinapeEpochTransformError(
+      'No member appears to have participated in this epoch; total GIVE given is zero.',
+    )
+  }
+
+  return contributions.reduce((data, contribution) => {
+    const { address: downcaseAddress, received, sent } = contribution
+
+    // Coordinape CSV addresses are downcased, we prefer Checksum
+    const address = utils.toChecksumAddress(downcaseAddress)
```

Comment:
@digidigo — Would you believe this? Somehow, I’d have thought Coordinape would be a Checksum crew. I guess not.